portfolio/src/main/java/com/google/sps/data/BigfootSighting.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+public class BigfootSighting {
+  
+  private double lat;
```

Comment:
nit: we tend to avoid non-standard abbreviations (I think usually it would be lat/long but long is a keyword; might as well just spell out the whole thing).